unlock-app/src/components/interface/locks/CheckoutUrl/elements/LocksForm.tsx

```diff
@@ -300,6 +300,17 @@ export const LocksForm = ({
     return orderedLocks
   })
 
+  useEffect(() => {
```

Comment:
This is required because the locks may be missing when we load the component for the first time